lib/core/jdl_entity.js

```diff
@@ -32,6 +32,7 @@ class JDLEntity {
     this.tableName = merged.tableName || merged.name;
     this.fields = merged.fields;
     this.comment = merged.comment;
+    this.options = merged.options;
```

Comment:
What are options exactly? Strings or objects? If they are strings, why not have a Set?

Reply:
It depends on where you are checking, and I totally agree it's a bit messed up, but if you check `jdl_options.js` (and I didn't touch that file), options are stored in an object (used as a Map), but when getOptions is called and array is returned. 
For places where this was the case (options did exist), it's still the sam (to avoid breaking changes in the public API). for places where I added options, I preferred keeping it simple and directly exposing the object.